cxgo/main.go

```diff
@@ -352,6 +384,24 @@ func main() {
 				log.Error(scanner.Text())
 			}
 		}()
+
+		if options.skipTrackerPing {
+			log.Info("Pinging CX Tracker is disabled")
+		} else {
+			log.Info("Initiating up CX Tracker ping")
+			pingTracker(options.configPath) // TODO should we ping on startup or after first period of 5 min is over and we're sure instance is running correctly?
```

Comment:
Not sure if we discussed this. Think we said it's ok to ping immediately but just to double check that statement.

Reply:
I think it's okay.